server/database/db.js

```diff
@@ -1,13 +1,16 @@
-require("dotenv").config();
-const { Pool } = require("pg");
-
-const pool = new Pool({
-  user: process.env.PG_USER,
-  host: process.env.PG_HOST,
-  database: process.env.PG_DATABASE,
-  port: process.env.PG_PORT,
-  password: process.env.PG_PASSWORD,
-  ssl: true,
-});
-
-module.exports = pool;
+// require("dotenv").config();
```

Comment:
Nit: can we avoid commenting out code. If we want to remove code the we should just ✂️